ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -610,18 +614,33 @@ module.exports = function OpenlayersMap(
     showMultiLineShape: function(locationModel) {
       let lineObject = locationModel
         .get('multiline')
-        .map(line => line.map(coords => convertPointCoordinate(coords)))
+        .map(line => line.map(coords => unconvertPointCoordinate(coords)))
+
+      const bufferedWidth =
+        DistanceUtils.getDistanceInMeters(
+          locationModel.get('lineWidth'),
+          locationModel.get('lineUnits')
+        ) || DEFAULT_BUFFER
```

Comment:
❓ 	Should we skip adding a buffer if none is provided instead of using a default?

Reply:
I made these consistent with how we handle buffers on other geometries. I also don't think a line with a zero buffer would have much significance in terms of searches